client/src/components/post/components/PostAuthorAction.jsx

```diff
@@ -16,6 +18,11 @@ const PostAuthorAction = ({ id, userId, text, visibility, firstName }) => {
       // invalidate getFeedPosts query to refetch it
       queryClient.invalidateQueries('getFeedPosts');
     },
+    onError: (error) => {
```

Comment:
This code snippet is repeated quite often. Create a function that accepts onError and navigate.

Reply:
done!